build/depends.py

```diff
@@ -1322,10 +1381,13 @@ def configure(self, build, conf):
             build.env.Append(CCFLAGS='/MP')
 
             # Generate debugging information for compilation units and
-            # executables linked regardless of whether we are creating a debug
-            # build. Having PDB files for our releases is helpful for debugging.
-            build.env.Append(LINKFLAGS='/DEBUG')
-            build.env.Append(CCFLAGS='/Zi /Fd${TARGET}.pdb')
+            # executables linked if we are creating a debug build or bundling
+            # PDBs is enabled.  Having PDB files for our releases is helpful for
+            # debugging, but increases link times and memory usage
+            # significantly.
+            if build.build_is_debug or build.bundle_pdbs:
```

Comment:
There's no need for this, since AppVeyor still builds with Qt 4, it'd be nice to have PDBs if we can.